README.md

```diff
@@ -57,7 +62,18 @@ When relevant, we encourage you to write tests. You can run the tests using the
 In addition to the code tests there are also visual regression tests, located in the `./cypress/integration` directory. You can run these tests locally by installing [cypress](https://www.cypress.io/) using `npm i cypress@3.0.3`, after which the command `npm run cypress` will run these tests locally. However, note that these tests are currently intended for screenshot comparisons across branches, and so will not yield any meaningful results when run for a single branch.
 
 
+## How to Setup your Dev Environment with Docker
+
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) (macOS and Windows). For Linux users: install [Docker CE](https://docs.docker.com/install/#supported-platforms) and [Docker Compose](https://docs.docker.com/compose/install/),
```

Comment:
Maybe include the direct links to the install or to the comments in this [docker-login thread](https://github.com/docker/docker.github.io/issues/6910#issuecomment-398819417)